pkg/scyllaclient/client_scylla_integration_test.go

```diff
@@ -75,17 +76,20 @@ func TestClientStatusIntegration(t *testing.T) {
 
 func TestClientDescribeRingIntegration(t *testing.T) {
 	testCases := []struct {
+		name            string
 		replicationStmt string
 		replication     scyllaclient.ReplicationStrategy
 		rf              int
 		dcRF            map[string]int
 	}{
 		{
+			name:            "simple_4",
```

Comment:
WDYT about randomizing the names by adding UUID for example ?

Reply:
I don't see a particular reason in the context of this test, as we won't have 2 tests with the same name here, but this could be an improvement used in all tests. We could introduce a function like:
```go
func getRandomizedName(name string) string {
    return name + "_" + fmt.Sprint(uuid.MustRandom)
}
```
That would be called anytime we want to obtain unique keyspace/table name.
But I would view it as a thing for a separate PR.